python/xorbits/_mars/tensor/datasource/tri.py

```diff
@@ -45,6 +45,7 @@ def tile(cls, op):
             yield
         tensor = op.outputs[0]
 
+        need_broadcast = False
```

Comment:
Only one-dimensional tensors require special treatment, so it can be named `is_1d_tensor`.